Bottom: 8e90cce39746205e383be705307af80cbe5cb017
Top:    4f3d293931fa91bad71e64e1774e528f75e4969c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:43:45 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
